share/sysbuild/cmake/modules/sysbuild_root.cmake

```diff
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024, Nordic Semiconductor ASA
+
+# Convert Zephyr roots to absolute paths to be used by sysbuild.
+#
+# This CMake module will convert all relative paths in existing ROOT lists to
+# absolute path relative from APP_DIR.
+#
+# Optional variables:
+# - ARCH_ROOT:       CMake list of arch roots containing arch implementations
+# - SOC_ROOT:        CMake list of SoC roots containing SoC implementations
+# - BOARD_ROOT:      CMake list of board roots containing board and shield implementations
+# - MODULE_EXT_ROOT: CMake list of module external roots containing module glue code
```

Comment:
`SCA_ROOT` is missing (it was also missing from `root.cmake` though)

Reply:
added both places (here and in root.cmake).